src/driver/PostgresDriver.ts

```diff
@@ -406,13 +406,13 @@ export class PostgresDriver implements Driver {
 
         const builtParameters: any[] = [];
         const keys = Object.keys(parameters).map(parameter => "(:" + parameter + "\\b)").join("|");
-        sql = sql.replace(new RegExp(keys, "g"), (key: string) => {
+        sql = sql.replace(new RegExp(keys, "g"), (key : string,  ...args : any[] ) : string  => {
```

Comment:
there is no in a comma between variable name and type definition. Also `...args : any[]` is optional, and there is no need in it